hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/pipeline/Pipeline.java

```diff
@@ -196,6 +208,7 @@ public boolean isEmpty() {
         .setFactor(factor)
         .setState(PipelineState.getProtobuf(state))
         .setLeaderID(leaderId != null ? leaderId.toString() : "")
+        .setCreationTimeStamp(creationTimestamp)
```

Comment:
creationTimestamp de-serialization is missed in getFromProtobuf.  
And the Builder doesn't support creationTimestamp.

Reply:
Updated.